### PR TITLE
Allow .csv files to have blank lines when viewed in Excel

### LINKED ISSUE #48: Make input decks handle blank lines correctly
### Desired capability or behavior.

Users (like Jes! thanks Jes) sometimes will have blank lines in their .csv file for readability. We should be able to handle this. Shouldn't take a huge amount of time to change this in `create_vehicle`.

### Is your feature request related to a problem? Please describe.

_No response_

### Associated Bug Report

_No response_

### PR DESCRIPTION
**Pull request by johnjasa**
### Summary

Users might edit .csv files in Excel and have visually blank lines.
These blank lines actually contain just commas from Excel, like this: ",,,,"
This PR allows Aviary to correctly read in files like these and adds a test accordingly.

### Related Issues

- Resolves #48

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/utils/process_input_decks.py [hunk 1]
```diff
@@ -73,6 +100,11 @@ def parse_inputs(vehicle_deck, aircraft_values: AviaryValues(), meta_data=_MetaD
 
             # remove any elements that are empty (caused by trailing commas or extra commas)
             data_list = [dat for dat in data.split(',') if dat != '']
+
+            # continue if there's no data in the line but there are commas
```
Comment (Kenneth-T-Moore) [inline aviary/utils/process_input_decks.py:104]: Since we can process these line-by-line, maybe we could add support for comments too (line startswith #)

**Review by Kenneth-T-Moore — APPROVED** [review]

File: aviary/utils/process_input_decks.py (hunk 1 above)
Comment (crecine) [reply aviary/utils/process_input_decks.py:104]: Lines 94-99 do that

Comment (crecine) [reply aviary/utils/process_input_decks.py:104]: actually, we might be able to remove 98 and 99?

File: aviary/utils/process_input_decks.py [hunk 2]
```diff
@@ -105,6 +137,19 @@ def parse_inputs(vehicle_deck, aircraft_values: AviaryValues(), meta_data=_MetaD
...
 
 def update_options(aircraft_values: AviaryValues(), initial_guesses):
+    """
+    Updates options based on the current values in aircraft_values. This function also handles special cases 
+    and prints debug information if the debug mode is active.
+
+    Parameters
+    ----------
+    aircraft_values (AviaryValues): An instance of AviaryValues containing current aircraft values.
+        initial_guesses (dict): A dictionary of initial guesses for various parameters.
```
Comment (crecine) [inline aviary/utils/process_input_decks.py:147]: extra tab?

**Review by crecine — APPROVED** [review]

File: aviary/utils/process_input_decks.py (hunk 2 above)
Comment (johnjasa) [reply aviary/utils/process_input_decks.py:147]: thanks, fixed

File: aviary/utils/process_input_decks.py (hunk 1 above)
Comment (johnjasa) [reply aviary/utils/process_input_decks.py:104]: we need 98-99 for when lines contain only commas. I updated the in-line comment

**Pull request merged**